src/pages/Home.jsx

```diff
@@ -58,18 +57,6 @@ margin:auto;
  padding: 2rem;
  border-radius: 8px;
 `;
-const NotesHeader = styled.div`
-display: flex;
-align-items: center;
-border-bottom: 1px solid var(--grey-border);
-max-width: 100%;
-position: sticky;
-
-@media(max-width:550px){
-  flex-direction: column;
-  align-items: flex-start;
-}
-`;
 const HeadingLarge = styled.h1`
 margin-right: auto;
 `;
```

Comment:
Suggestion: You can store this in utilities file, since you are useing the same variable in multiple places(archive page, home page and trash page)

Reply:
thank you moved this component and few others to utilities folder